configs/sites/noaa-azure/README.md

```diff
@@ -6,7 +6,11 @@ that the site configuration files in `configs/sites/noaa-azure` work out of the
 
 sudo su
 chmod 777 /contrib
-yum install -y m4
```

Comment:
Does the new installation work successfully without m4?

Reply:
oops that's a mistake I think. Glad you are reviewing thoroughly!

Reply:
fixed